test.js

```diff
@@ -1,7 +1,12 @@
-'use strict';
-var assert = require('assert');
-var splitLines = require('./');
+import test from 'ava';
+import fn from './';
```

Comment:
We never managed to agree on anything, so I just continued using `m` as I could use the same even if a module doesn't export a function. I like consistency. Happy to discuss. Maybe we should take it to https://github.com/sindresorhus/meta